guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -94,7 +94,15 @@ Reloading the page only triggers compilation and publication of the styles used
 
 ##### Debugging using the static view files deployment tool
 
-Once you save your changes and run the `magento setup:static-content:deploy` cli command from the `<your_Magento_instance>/bin` directory, the tool pre-processes (including compilation) and publishes the static view files.
+Once you save your changes, run the following command from your `<Magento_root>` directory:
+
+```bash
+bin/magento setup:static-content:deploy
+``` 
+The tool pre-processes (including compilation) and publishes the static view files.
+
+{:.bs-callout .bs-callout-info}
+NOTE: Manual static content deployment is not required in "default" and "developer" modes. If you still want to deploy in these modes, use the -f option: `bin/magento setup:static-content:deploy -f`
```

Comment:
I don't think we need to add the "NOTE:" because it's in the callout.

Reply:
Good point.